Code/Framework/AzToolsFramework/AzToolsFramework/AssetEditor/AssetEditorBus.h

```diff
@@ -8,12 +8,17 @@
 #pragma once
 
 #include <AzCore/Asset/AssetCommon.h>
+#include <AzCore/Outcome/Outcome.h>
 #include <AzCore/UserSettings/UserSettings.h>
-#include <AzCore/Asset/AssetCommon.h>
 #include <AzCore/std/containers/unordered_map.h>
-#include <AzCore/Serialization/SerializeContext.h>
 
-namespace AZ { namespace Data { class AssetData; } }
+namespace AZ
+{
+    namespace Data
+    {
+        class AssetData;
+    }
+} // namespace AZ
```

Comment:
This can use the C++17 nested namespace feature
```suggestion
namespace AZ::Data
{
    class AssetData;
}
```

Reply:
Indeed they can 😄 
committed suggested edit in c027336